### DISCUSSION TITLE
Open Reel

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#156] [Presets]
This preset mimics the 1/2" Open Reel tape format. This was used as an example in my [Guide to Presets](https://github.com/valadaptive/ntsc-rs/discussions/140)

![image](https://github.com/user-attachments/assets/ab16f594-08fb-43b1-8be4-9a8ccf70fce6)

[OpenReel.json](https://github.com/user-attachments/files/16562760/OpenReel.json)

**Comment by smfthe1199** [comment]
Immediately added to this preset pack.

Btw, in which category do you think this will go to?

Edit: nvm, it will be on the _"VHS Tapes"_ category.

**Reply by Degamisu** [reply]
Since VHS tapes are completely seperate to open reel, I think the category should be renamed to Tapes. (Just a suggestion!)

**Reply by smfthe1199** [reply]
Ok..
